nemo/collections/nlp/models/language_modeling/megatron_gpt_sft_model.py

```diff
@@ -93,6 +94,9 @@ def __init__(self, cfg: DictConfig, trainer: Trainer):
         self.original_checkpointing_num_layers = base_module.language_model.encoder.activations_checkpoint_num_layers
         self.original_checkpointing_method = base_module.language_model.encoder.activations_checkpoint_method
         self.virtual_tokens = 0
+        # Create a different self.validation_step_outputs as we call val step of parent class(MegatronGPTModel) here,
+        # that appends to self.validation_step_outputs leading to wrong values in self.validation_step_outputs
+        self.validation_step_outputs_sft = []
```

Comment:
Do we need both `self.validation_step_outputs` (from the parent) and `self.validation_step_outputs_sft` (from the child)?

Reply:
Fixed and discussed offline.